framework/src/service/ServiceRegistrationBase.cpp

```diff
@@ -175,27 +160,27 @@ namespace cppmicroservices
                 }
             }
 
-            auto oldRankAny = d->properties.Value_unlocked(Constants::SERVICE_RANKING).first;
+            auto oldRankAny = d->coreInfo->properties.Value_unlocked(Constants::SERVICE_RANKING).first;
             if (!oldRankAny.Empty())
             {
                 // since the old ranking is extracted from existing service properties
                 // stored in the service registry, no need to type check before casting
                 old_rank = any_cast<int>(oldRankAny);
             }
-            d->properties = Properties(AnyMap(std::move(propsCopy)));
+            (d->coreInfo->properties) = Properties(AnyMap(std::move(propsCopy)));
```

Comment:
Are these parentheses necessary?

Reply:
No they aren't, thanks!

Updated in "Updated based on Patty's comments"(https://github.com/CppMicroServices/CppMicroServices/pull/841/commits/7d7bdd60e332b54a5954fe8823ccc2212b5a46d3)